javalib/src/main/scala/java/util/AbstractList.scala

```diff
@@ -1,3 +1,244 @@
 package java.util
 
-abstract class AbstractList[E] extends AbstractCollection[E] with List[E]
+import java.util.{Iterator => JIterator}
+
+import scala.annotation.tailrec
+
+abstract class AbstractList[E] protected extends AbstractCollection[E] with List[E] {
```

Comment:
Write explicitly `()` for the constructor when it has a modifier, like this: `protected () extends`. Otherwise the `protected` keyword there is very confusing.